src/app/page/MissionCreate/MissionCreate.jsx

```diff
@@ -76,17 +74,26 @@ function MakeMission({ history }) {
       Mission.create(mission)
         .then(() => {
           setLoading(false);
-          setSuccessSnackbarOpen(true);
+
+          snackbarContext.updateSnackbar({
```

Comment:
maybe instead of having updateSnackbar, we could just have snackbarContext.show(....) ?

Reply:
I can update the method name

Reply:
Done :+1: